script/caffe2intel-i8/finalize_train_val_prototxt.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (c) 2018 cTuning foundation.
+# See CK COPYRIGHT.txt for copyright details.
+#
+# SPDX-License-Identifier: BSD-3-Clause.
+# See CK LICENSE.txt for licensing details.
+#
+
+import argparse
+import os
+
+from caffe.proto import caffe_pb2
+from google.protobuf import text_format  
+
+if __name__ == '__main__':
+  parser = argparse.ArgumentParser()
+  parser.add_argument('-m', '--model', action='store', dest='MODEL')
+  params = parser.parse_args()
+
+  print('Postprocessing {} ...'.format(params.MODEL))
+
+  net = caffe_pb2.NetParameter()
+  with open(params.MODEL, 'r') as f:
+    text_format.Merge(f.read(), net)
+
+  # Adjust input layer
+  # Currently it's supposed that there is only one input layer
```

Comment:
NB: Currently, we assume that there is only one input layer.

Reply:
Changed.